resources/js/components/FormField.vue

```diff
@@ -72,10 +72,19 @@
 						this.dependenciesSatisfied = false;
 						return;
 					}
-
-					if(dependency.hasOwnProperty('value') && this.dependencyValues[dependency.field] !== dependency.value) {
-						this.dependenciesSatisfied = false;
-						return;
+					
+					if(dependency.hasOwnProperty('values')) {
+						let valuesSatisfied = false;
+						for (let value of dependency.values) {
+							// Use coercion as form values are always string
+							if (this.dependencyValues[dependency.field] == value) {
+								valuesSatisfied = true;
```

Comment:
a nice optimization here might be to return as soon as we find a value satisfied. There is no point to check other conditions if one matches. Also you could instead use the js `includes` method to make this even cleaner instead of a local variable (`valuesSatisfied`) and the for loop

Reply:
@ragingdave that’s basically the only problem I have with this package. When chaining multiple `dependsOn` conditions, it will always resolve in an `or` statement, when dependencies get resolved as soon as one statement resolves to be true.

Reply:
While I agree with you, and might have even said so in an issue about this same topic, that level of improvement for this package seems a very far ways off. I mean this with no disrepect, but clearly you don't have the time for that big of an upgrade, and quite frankly neither do I. Perhaps this particular PR, would be valuable to further improve the 1.x line, with perhaps the query builder like syntax/chaining being something for a v2 or beyond.